stubs/Extensions/Process+Stubs.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
```

Comment:
Very simple extension that adds the ability to execute an arbitrary command from bash.